unison-src/transcripts/delete-namespace.output.md

```diff
@@ -89,7 +90,12 @@ scratch/main> delete.namespace .
 -- Should have an empty history
 scratch/main> history .
 
-  ☝️  The namespace . is empty.
+  Note: The most recent namespace hash is immediately below this
+        message.
+  
+  
+  
+  □ 1. #sg60bvjo91 (start of history)
```

Comment:
Notably you can no longer have an actually "empty" history, the project-branch-head has to be set to something, in this case the empty branch.

We could possibly check for equivalence to the empty hash and call it out in the output if we want.